### PR TITLE
Added 4.6 channel

### PR DESCRIPTION
**Pull request by slintes**
and updated operator update test to run from 4.5. to 4.6

### CODE REVIEW

**Comment by slintes** [comment]
/hold

the upgrade lane reports green, but it has a test failure
investigating the failure and the wrong report

**Comment by slintes** [comment]
failure report should be fixed now

**Comment by slintes** [comment]
friendly reminder: please update the channel in the feature deploy repo once this is merged:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/deploy/performance/operator_subscription.yaml#L7

@fedepaol @cynepco3hahue fyi

File: functests-extended/1_performance_operator_upgrade/upgrade_operator.go [hunk 1]
```diff
@@ -17,45 +18,52 @@ import (
...
+		By(fmt.Sprintf("Verifying that channel was updated to %s", toVersion))
+		subscriptionWaitForUpdate(subscription.Name, operatorNamespace, toVersion)
 
 		// CSV is updated and pointed to right image tag
 		subscription = getSubscription(subscriptionName, operatorNamespace)
 		csv = getCSV(subscription.Status.CurrentCSV, operatorNamespace)
 		csvWaitForPhaseWithConditionReason(csv.Name, operatorNamespace, olmv1alpha1.CSVPhaseSucceeded, olmv1alpha1.CSVReasonInstallSuccessful)
-
-		// CRD should be current version with all new features
-		crd = getCRD(csv.Spec.CustomResourceDefinitions.Owned[0].Name)
```
Comment (dshchedr) [inline functests-extended/1_performance_operator_upgrade/upgrade_operator.go:57]: hm, as I see we don't have CRD verification.. I'm not sure we are really ok with that, but if we don't need it - we can remove unused function: getCRD()

Comment (slintes) [reply functests-extended/1_performance_operator_upgrade/upgrade_operator.go:57]: There is nothing we can test yet, 4.5 and 4.6 CRDs are the same AFAIK.
Removed unused func, added test id

**Comment by slintes** [comment]
rebased... no clue why e2e fails all the time with the same 2 failures

File: functests-extended/1_performance_operator_upgrade/upgrade_operator.go (hunk 1 above)
Comment (yanirq) [reply functests-extended/1_performance_operator_upgrade/upgrade_operator.go:57]: proably it should be included in https://github.com/openshift-kni/performance-addon-operators/pull/249 or in a following PR after merge.

File: deploy/olm-catalog/performance-addon-operator/4.6.0/performance-addon-operator.v4.6.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: performance-operator
+                image: REPLACE_IMAGE
```
Comment (yanirq) [inline deploy/olm-catalog/performance-addon-operator/4.6.0/performance-addon-operator.v4.6.0.clusterserviceversion.yaml:129]: Are we getting the right image here ?

**Comment by yanirq** [comment]
> rebased... no clue why e2e fails all the time with the same 2 failures

In previous test runs i saw old kernel arguments values, is there a change the 4.6 is not replace correctly ?  
I added a fix in a PR for one of the kernel argument. We can compare the test run results there when they are completed.

**Comment by slintes** [comment]
> In previous test runs i saw old kernel arguments values (systemd.cpu_affinity is missing), is there a chance the 4.6 image is not replace correctly ?

damn, I forget to update the channel in the subscription...
fixed...

File: deploy/olm-catalog/performance-addon-operator/4.6.0/performance-addon-operator.v4.6.0.clusterserviceversion.yaml (hunk 2 above)
Comment (slintes) [reply deploy/olm-catalog/performance-addon-operator/4.6.0/performance-addon-operator.v4.6.0.clusterserviceversion.yaml:129]: yes, it is replaced with the image from CI during image build, see https://github.com/openshift-kni/performance-addon-operators/blob/master/openshift-ci/Dockerfile.registry.build#L12

**Comment by slintes** [comment]
e2e tests succeeded, however the lane is red for some reason

/retest

@dshchedr can you please lgtm again :)

**Pull request merged**